test/parallel/test-cluster-shared-leak.js

```diff
@@ -15,6 +15,11 @@ if (cluster.isMaster) {
   worker1 = cluster.fork();
   worker1.on('message', common.mustCall(function() {
     worker2 = cluster.fork();
+    worker2.on('error', function(e) {
+      // EPIPE is OK on Windows
+      if ((! common.isWindows) || (e.code !== 'EPIPE'))
```

Comment:
Aside from some style issues, would an opposite check be more clear? i.e. `if (common.isWindows && e.code === 'EPIPE') return;`?